FlowCal/excel_ui.py

```diff
@@ -133,7 +141,8 @@ def read_table(filename, sheetname, index_col=None):
 
     """
     # Catch sheetname as list or None
-    if sheetname is None or hasattr(sheetname, '__iter__'):
+    if sheetname is None or \
+        (hasattr(sheetname, '__iter__') and not isinstance(sheetname, str)):
```

Comment:
I'm used to checking `isinstance(other, basestring)`.

This [Stack Overflow post](https://stackoverflow.com/questions/4843173/how-to-check-if-type-of-a-variable-is-string) seems to suggest that checking for a string is different in Python 2.7 versus Python 3.*. They also mention using the `six` module if you want cross-Python version compatibility? Did you already look into this? I think this ultimately boils down to the current code won't detect `unicode` objects as strings? (not sure if we care about that)